parameters_meta_schema.json

```diff
@@ -151,5 +224,24 @@
     "title",
     "description",
     "type"
-  ]
+  ],
+  "$defs": {
+    "typeAnnotation": {
+      "type": "string",
+      "enum": [
+        "string",
+        "boolean",
+        "integer",
+        "number",
+        "null"
+      ]
+    },
+    "allTypes": {
+      "type": ["integer", "boolean", "number", "string", "null"]
+    },
```

Comment:
How is this different to the one above?

Reply:
The above one is a definition that states the value should be a string of one of these: "string", "boolean", "integer", "number" or "null". 
The bottom one states that value should be one of those types. (It's just a shortcut I wrote to keep the redundancy as low as I could)

Reply:
Gotcha 👍🏻 Can you reference this above as well so that there's only one set, out of interest? I guess not as it needs a `$ref` key and that'd break the array structure.. 🤔

Reply:
I'm not sure I'm following what you mean here :grin: